### ISSUE TITLE
[Suggestion] Just an easy solution to duplicate cards

### ISSUE THREAD

**Issue by zedekblue** [jgoldshlag/padherder_proxy#33]
Hey, so I do a lot of collecting. Probably why you've heard from me so much around here. PHP is a lifesaver when keeping track of everything I have.   
  
However, I've had a lot of issues recently with the proxy *technically* deleting the wrong card.  
I use Padherder's notes to keep track of what material I'm holding onto, and for what reason. Example:  
  

![example](https://cloud.githubusercontent.com/assets/17040113/20249370/1271c1aa-a9b5-11e6-8f12-7152e3a6a1ec.png)
  
However, when these cards are actually used, I generally have one or two of them lying around. The proxy obviously doesn't know which material I've assigned to which card, and suddenly I'm short a Canopus, and holding onto an extra Star Justice for no good reason.  
  
Long story short, I just thought it'd be really cool to add a toggle for the proxy to prompt the user before removing anything. If any duplicates are found, let the user choose which to remove, and display any notes or information associated with it.
